contribs/gmf/less/desktoplayertree.less

```diff
@@ -35,8 +35,9 @@ gmf-layertree {
 
   &.dragger {
     * {
-      cursor: grabbing;
+      cursor: url(../cursors/grabbing.cur);
```

Comment:
the fallback is missing, must be: `cursor: url(../cursors/grabbing.cur), default;`